app/models/linked_case.rb

```diff
@@ -0,0 +1,74 @@
+class LinkedCase < ApplicationRecord
```

Comment:
Can you run `bundle exec annotate` so that we get a list of columns?

Reply:
Yes!